Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,612 @@
+# Frequenty asked questions
+
+## General information and availability
+
+### Do I need VTK to build ITK ?
+
+With the default configuration, ITK does not require VTK. However, some
+optional modules (e.g. [ITKVTKGlue](https://github.com/InsightSoftwareConsortium/ITKVtkGlue)) do require VTK.
+
+ITK does not provide any visualization functionalities. Applications
+requiring visualization should implement those functionalities from
+internal code or libraries like OpenGL, GLUT or VTK.
+
+### What 3D file formats can ITK import and export?
+
+The following table identifies the file formats that ITK can read/write
+via a factory mechanism based on the file extension
+([`itk::ImageIOFactory`](https://itk.org/Doxygen/html/classitk_1_1ImageIOFactory.html)). Some proprietary files format are only imported.
+More information on [ITK File Formats](https://itk.org/Wiki/ITK_File_Formats) is
+available, including data types supported.
+
+A very nice tool for viewing all of these images is ImageViewer which
+can be found in the ITKApps repository.
+
+| File Format | Read/Write | Import |
+| ----------- | ---------- | ------ |
+| Analyze 7.5 | AnalyzeImageIO | |
+| BMP † | BMPImageIO | |
+| DICOM | GDCMImageIO | Using DicomImageIO is backward compatible, but user should prefer GDCMImageIO |
+| GE 4x | | GE4ImageIO |
+| GE 5x | | GE5ImageIO |
+| GE Advantage Windows | | GEAdwImageIO |
+| Gipl (.gipl) | GiplImageIO | |
+| IPLCommon | | IPLCommonImageIO |
+| JPEG † | JPEGImageIO | |
+| NRRD (.nhdr/.nrrd) | NrrdImageIO | |
+| Nifti (.nii) | NiftiImageIO | |
+| MetaImage (.mha/.mhd) | MetaImageIO | |
+| PNG † | PNGImageIO | |
+| Raw ‡ | RawImageIO | |
+| Siemens Vision | | SiemensVisionImageIO |
+| Stimulate (spr/sdt) | StimulateImageIO | |
+| TIFF | TIFFImageIO | |
+| VTK Structured Points | VTKImageIO | |
+
+† BMP, PNG and JPEG are not very good formats for 3D medical images.
+They only support 2D images and a limited set of pixel types such as
+unsigned char, and unsigned short. The great advantage of BMP, PNG and
+JPEG is that you can view them with almost any image viewer. It is
+possible to read/write 3D datasets as collections of 2D slices each one
+in an independent BMP, PNG or JPEG file by using the
+`ImageSerierReader` and [`itk::ImageSeriesWriter`](https://itk.org/Doxygen/html/classitk_1_1ImageSeriesWriter.html).
```

Comment:
`ImageSerierReader` does not exist as a class.

Reply:
I think it should be `ImageSeriesReader`.

Reply:
🤦. Done in 894e5e6.